drivers/sensor/Kconfig.sensor_clock

```diff
@@ -0,0 +1,29 @@
+# Sensor clock configuration options
+# Copyright(c) 2024 Cienet
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig SENSOR_CLOCK
```

Comment:
I just realized, this shouldn't be selectable. Please remove the short description "Sensor clock configuration" and add under the bool:

```
default y if SENSOR_ASYNC_API
```

Reply:
Done